engine/flink/executor/src/test/scala/pl/touk/nussknacker/engine/process/typeinformation/TypingResultAwareTypeInformationDetectionSpec.scala

```diff
@@ -236,32 +220,6 @@ class TypingResultAwareTypeInformationDetectionSpec
       .isIncompatible shouldBe true
   }
 
-  test("serialization compatibility with reconfigured serializer") {
```

Comment:
This test used a deprecated API and shows a situation that won't happen anymore because we won't change kryo serializers setups anytime - it will be constant